src/loadcore/hash.c

```diff
@@ -16,7 +16,7 @@
  * 
  * Returns the index in the hash table where the array can be found
  */
-__inline__ u32 getCyclicPolynomialHash(const char *str, u32 radix, u32 hashTableSize)

```

Comment:
In the ASM, this function has been inlined. How will removing this keyword here affect the compiler output?

Reply:
The issue was with the export that occurs for this as well. Will take another look at it.

Reply:
Ah, this error comes up when `__inline__` is specified in the function definition in the header
```
In file included from loadcore.c:43:
hash.h:14:17: error: inline function ‘getCyclicPolynomialHash’ declared but never defined [-Werror]
   14 | __inline__  u32 getCyclicPolynomialHash(const char *str, u32 radix, u32 hashTableSize);```